var/spack/repos/builtin/packages/go/package.py

```diff
@@ -82,10 +82,17 @@ def patch(self):
 
     def install(self, spec, prefix):
         bash = which('bash')
-        with working_dir('src'):
+
+        wd = '.'
+
+        # 1.11.5 directory structure is slightly different
+        if self.version == Version('1.11.5'):
```

Comment:
We could assume this will be the case going forward and add `if self.version >= Version('1.11.5')`

Reply:
I think this is just a mistake in the 1.11.5 release. It has some empty top-level directories in the tar.gz which prevent Spack from moving the source directory up one level (cf. fetch_strategy.py:339).

Reply:
Got it - thanks for the clarification